docs/styles/prism-synthwave-84.css

```diff
@@ -36,6 +36,11 @@ pre[class*='language-'] {
   overflow: auto;
 }
 
+/* Plaintext code blocks */
+:is(code,pre)[class='language-plaintext'] {
+  color: #fff5f6;
+}
+
```

Comment:
In some places, I wanted to show a sample tree view for a project. Fenced code blocks with no language looked a bit ugly, so I opted instead for the approach in 459f3d4. It's not ideal because we need to remember to add `plaintext` to code blocks, but it works:

![image](https://user-images.githubusercontent.com/19352442/149239772-61d649ab-e0a1-46f7-bbaa-6c4b90a52cd8.png)

On my site, so that I don't have to remember to add `plaintext` for plaintext code blocks, I opted to remove 11ty's syntax highlighter plugin and replace it with markdown-it-prism, setting the default language to `plaintext`:

https://github.com/AleksandrHovhannisyan/aleksandrhovhannisyan.com/blob/master/config/plugins/markdown.js#L15

Thoughts on doing this?

Reply:
I use the `tree` command in Bash for this (`tree -I node_modules`).

Not sure if the extra characters add unnecessary noise for screen readers, though; I haven't tested it. I could use indentation here if you'd like—up to you!

Reply:
I suppose we could also just override the colors for `language-bash` and use that instead.

Reply:
Yes to `plaintext` as a "default." I mainly used `bash` to add comments in-line where necessary. But for file trees like this, I agree with you

Reply:
Sounds good, I'll do that as a separate PR to keep this one more manageable

Reply:
Awesome! I'll also get this in once I add an initial draft for "how we build your site." Prioritizing work on renderers right now but it's on my todos for the extended weekend ❤️